src/coreComponents/common/unitTests/testLifoStorage.cpp

```diff
@@ -0,0 +1,160 @@
+#include "mainInterface/initialization.hpp"
+#include "common/lifoStorage.hpp"
+#include "LvArray/src/Array.hpp"
+#include "LvArray/src/MallocBuffer.hpp"
+#if defined(LVARRAY_USE_CHAI)
+  #include "LvArray/src/ChaiBuffer.hpp"
+#endif
+
+#include <gtest/gtest.h>
+
+#ifndef __CUDA_ARCH__
```

Comment:
This is all copied from somewhere right?

Reply:
Yes I started from another test, I don't remember which one.